### PR TITLE
Update collections.c

### PR DESCRIPTION
**Pull request by NeelamAggarwal**
Segmentation fault. Corrected by removing the line cass_cluster_set_protocol_version(cluster, 2);

### CODE REVIEW

File: examples/collections/collections.c [hunk 1]
```diff
@@ -165,7 +165,7 @@ int main(int argc, char* argv[]) {
 
   const char* items[] = { "apple", "orange", "banana", "mango", NULL };
 
-  cass_cluster_set_protocol_version(cluster, 2);
+  //cass_cluster_set_protocol_version(cluster, 2);
```
Comment (mpenick) [inline examples/collections/collections.c]: Thanks! This could be removed instead of commented (I don't know why that was in there).

**Comment by mpenick** [comment]
Thanks for the commit. Could you please sign the CLA?

**Comment by NeelamAggarwal** [comment]
I signed the CLA before. Thanks.

**Comment by avalanche123** [comment]
@NeelamAggarwal you haven't signed the CLA until you've done it in Docusign, can you please make sure you complete it?

**Comment by mpenick** [comment]
@NeelamAggarwal Can you please remove the line (instead of commenting)? Then I'll pull it in. Thanks!

**Comment by NeelamAggarwal** [comment]
I did. Please check. Thank you. :)

**Comment by mpenick** [comment]
Merged @ 9c13e71ee2f91414be28e65744dfe419db460c81. Thanks.

**Pull request closed without merging**